src/scripts/tests/test-create-eventbridge-rule.ts

```diff
@@ -0,0 +1,52 @@
+import * as EventBridge from '@aws-sdk/client-eventbridge';
+import { fromIni } from '@aws-sdk/credential-provider-ini';
+import assert from 'assert';
+import console from 'console';
+import * as Luxon from 'luxon';
+import process from 'process';
+import { putRule, putTarget } from '../../lib/create-eventbridge-rule';
+import { generateCronExpressionUtc } from '../../lib/cron-utils';
+import { Reservation } from '../../lib/reservation';
+import * as Queue from '../../lib/scheduled-checkin-ready-queue';
+
+async function main() {
+  assert(process.argv.length === 3, 'Invalid parameters');
```

Comment:
Does this assert happen after everything executes? Does this create resources in your AWS account that needs to be cleaned up eventually? 

Still new with AWS work and interested in how testing generally works in it!

Reply:
The assert is executed first.

Yes, this script will create an EventBridge rule and target. I suppose we could create an entire dev version of this stack for testing, but that doesn't seem worth it.